overlays/nodejs/autoscaling/notificationType.ts

```diff
@@ -0,0 +1,19 @@
+// This file contains two categories of exports:
+//
+//     1) A union type, NotificationType, that accepts any valid AutoScaling Notification type
+//     2) Individual constants for each such AutoScaling Notification type.
+//
+// These just give a better developer experience and are just sugared strings.
```

Comment:
Could we add a link to primary source in AWS docs for where these strings are defined?